lib/yearRange.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+/* eslint-disable */
+
+/**
+ * Function to generate a span of years based on minimum and maximum parameters
+ *
+ * @param {Object} input - inputs in this plugin's instance and their values as {String}
+ * @param {Object} minYear - inputs minimum year value {String}
+ * @param {Object} maxYear - inputs maxmium year value {String}
+ * *
+ * @module yearRange
+ */
+
+module.exports = function yearRange(input, minYear, maxYear) {
```

Comment:
As we talked about, UI scripts will have `inputs` and `settings` passed to it, and only those two things. `minYear` and `maxYear` are not things that will be passed to this function